argparse_prompt.py

```diff
@@ -64,20 +68,36 @@ def __init__(self, name=None, help=None, type=None, default=None, secure=False):
         self.help = help
         self.default = default
         self.secure = secure
+        self.choices = choices
 
     def __call__(self, val):
         default_str = f"({self.default}) "
         help_str = "" if self.help is None else f": {self.help}"
+        
+        # Make our choices pretty for display
+        choices = self.choices
+        if type(choices) == list:
+            choices_options ="({options}): ".format(options='|'.join(choices))
+        else:
+            choices_options = False
 
         try:
             # If the user provided no value for this argument, prompt them for it
             if val == "":
-                prompt = "{}{}\n> {}".format(self.name, help_str, default_str)
+                prompt = "{}{}\n> {}".format(self.name, help_str, choices_options)
```

Comment:
By removing `default_str` here it looks like this breaks the default prompt for non-choice args